starter-code/scripts/article.js

```diff
@@ -16,6 +16,10 @@ Article.prototype.toHtml = function() {
   //       - Get your template from the DOM.
   //       - Now "compile" your template with Handlebars.
 
+  var $source   = $('#blogTemplate').html();
+  var template = Handlebars.compile($source);
+  console.log(template);
```

Comment:
Though its nice to test things out with console logs, ensure that you do not push unnecessary code up to github. I imagine this console log is pretty big haha.